linux/keyman-config/keymankeyboards.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/python3
+
+# Read in local keyboards json file
+
+# Later check if online
+# If so then get it from web api
+# Is there a way to check if has changed from cached version and only download if it has?
+
+# Smaller than languages file (~60% of size) so more network efficent
+# but needs a little more processing
+# using gzip makes network transfer much less
+
+# for each keyboard
+# for each language in keyboard
+# add language to a region if it doesn't already exist
+# add keyboard to that language if it isn't already on it
+
+import datetime
+import time
+import json
+import requests
+import requests_cache
+import os
+
+def get_api_keyboards(verbose=False):
+	"""
+	Get Keyboards data from web api.
+
+	Args:
+		verbose(bool, default False): verbose output
+	Returns:
+		dict: Keyboard data
+		None: if http request not successful
+	"""
+	api_url = "https://api.keyman.com/cloud/4.0/keyboards"
```

Comment:
I think we only want keyboards for the `device` = `any` or `desktop`.

Maybe something similar to Keyman Desktop 
https://keyman.com/go/desktop/10.0/download-keyboards?version=...

Reply:
This was an experimental file so is still in git for further experiments but is not distributed